pkgs/tools/admin/google-cloud-sdk/default.nix

```diff
@@ -53,8 +53,8 @@ in stdenv.mkDerivation rec {
         programPath="$out/google-cloud-sdk/bin/$program"
         binaryPath="$out/bin/$program"
         wrapProgram "$programPath" \
-            --set CLOUDSDK_PYTHON "${python}/bin/python" \
-            --prefix PYTHONPATH : "${pythonPath}"
+            --set CLOUDSDK_PYTHON "${pythonEnv}/bin/python" \
+            --prefix PYTHONPATH : "${pythonEnv}/${python.sitePackages}"
```

Comment:
Does it really need `PYTHONPATH`? We already point the the interpreter, is that not enough?

Reply:
Their wrapper does a bunch of mangling with the execution environment. I can try with out it. I need some feedback from the actual users of the tool to see if it still works afterwards.